### DISCUSSION TITLE
How do you get rid of this red wavy line?

### DISCUSSION THREAD

**Discussion by 332lfgb6** [quasarframework/quasar#9487] [General - Components / Directives / etc]
![image](https://user-images.githubusercontent.com/51245335/120151936-76552b80-c21f-11eb-804b-ccf8a91ed7b3.png)
![image](https://user-images.githubusercontent.com/51245335/120151981-853bde00-c21f-11eb-9ee5-ec3d506a2e8f.png)
The procedure is normal
![image](https://user-images.githubusercontent.com/51245335/120152106-a7356080-c21f-11eb-97d1-c6dfa681c582.png)

**Comment by hawkeye64** [comment]
How did those get there? They are not in the starter kit (https://github.com/quasarframework/quasar-starter-kit/tree/next/template/src/css)
I think (AFAIK) it's safe to remove those. But, I also don't understand the intent.

**Reply by 332lfgb6** [reply]
After vue add quasar, it automatically generates two files (quasar.scss and quasar.varianles.scss) in the styles folder, and these two wavy lines are in these two files.
![image](https://user-images.githubusercontent.com/51245335/120253010-d1872c80-c2b8-11eb-92eb-0841fb8bfad9.png)
![image](https://user-images.githubusercontent.com/51245335/120253022-db109480-c2b8-11eb-8d1f-d7c1575a6e49.png)
These two wavy lines cause wavy lines in the file on the left, wavy lines in the Styles folder, and wavy lines in the project name.If there is no solution, I will skip it, because these warnings will not cause the program to report an error.
